mm/mm_heap/mm_memalign.c

```diff
@@ -146,11 +146,18 @@ FAR void *mm_memalign(FAR struct mm_heap_s *heap, size_t alignment,
     {
       FAR struct mm_allocnode_s *newnode;
       FAR struct mm_allocnode_s *next;
+      FAR struct mm_freenode_s *prev;
       size_t precedingsize;
 
+      /* Mark node free */
+
+      node->preceding &= ~MM_MASK_BIT;
+
       /* Get the node the next node after the allocation. */
 
       next = (FAR struct mm_allocnode_s *)((FAR char *)node + node->size);
+      prev = (FAR struct mm_freenode_s *)
+        ((FAR char *)node - node->preceding);
```

Comment:
I think if you out it in a single line it will be smaller than 78 chars

Reply:
I tried, it's 79 chars, checkpatch.sh report this:
```shell
wangbowen@ubuntu:~/project/community/nuttx_rpmsg_server/nuttx$ tools/checkpatch.sh -g HEAD~...HEAD
/home/wangbowen/project/community/nuttx_rpmsg_server/nuttx/mm/mm_heap/mm_memalign.c:159:78: error: Long line found
```